src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkWindows.cs

```diff
@@ -194,6 +194,17 @@ protected override IEnumerable<string> GetAllAvailableAndroidNdks ()
 
 			Logger (TraceLevel.Info, "Looking for Android NDK...");
 
+			// Check for the "ndk-bundle" directory inside the SDK directories
+			string ndk;
+
+			var sdks = GetAllAvailableAndroidSdks().ToList();
```

Comment:
Should this instead be:

```csharp
var sdks = new HashSet<string>(GetAllAvailableAndroidSdks (), StringComparer.OrdinalIgnoreCase);
sdks.Add (AndroidSdkPath);
```

Reply:
I was actually thinking on making the default sdk separate and then I don't need the ToList. The thing is all the users I could find of this method end up doing a First(). If I move the default sdk out and make it the first option, and if the sdk and ndk exists (which is always, when the ndk exists at all) then I will never hit the available sdks list (or anything else, for that matter). I'll submit a new PR for review with that change.